source/includes/_people.md

```diff
@@ -53,88 +53,159 @@ curl -X POST -H "Content-Type: application/json" -H "Cache-Control: no-cache" -d
             "organization_id": "5e66b6XXXXXXXXXXXXXXXXXX",
             "organization": {
                 "id": "5e66b6XXXXXXXXXXXXXXXXXX",
-                "name": "Apollo",
+                "name": "Apollo.io",
                 "website_url": "http://www.apollo.io",
                 "blog_url": null,
                 "angellist_url": null,
                 "linkedin_url": "http://www.linkedin.com/company/apolloio",
                 "twitter_url": "https://twitter.com/MeetApollo/",
                 "facebook_url": "https://www.facebook.com/MeetApollo/",
+                "primary_phone": {
+                    "number": "(202) 374-XXXX",
+                    "source": "Account"
+                },
                 "languages": [],
-                "alexa_ranking": 77520,
-                "phone": null,
-                "linkedin_uid": "18511550",
+                "alexa_ranking": 685,
+                "phone": "(202) 374-XXXX",
+                "linkedin_uid": "185115XX",
+                "founded_year": 2015,
                 "publicly_traded_symbol": null,
                 "publicly_traded_exchange": null,
-                "logo_url": "https://apollo-server.com/uploads/pictures/5f0265XXXXXXXXXXXXXXXXXX/picture",
+                "logo_url": "https://zenprospect-production.s3.amazonaws.com/uploads/pictures/6188c0967ae1c20001869130/picture",
                 "crunchbase_url": null,
                 "primary_domain": "apollo.io",
-                "starred_by_user_ids": [],
                 "persona_counts": {}
             },
-            "account_id": "5f1fadXXXXXXXXXXXXXXXXXX",
+            "account_id": "616d0eXXXXXXXXXXXXXXXXXX",
             "account": {
-                "id": "5f1fadXXXXXXXXXXXXXXXXXX",
+                "id": "616d0eXXXXXXXXXXXXXXXXXX",
                 "name": "Apollo",
                 "website_url": "http://www.apollo.io",
                 "blog_url": null,
                 "angellist_url": null,
                 "linkedin_url": "http://www.linkedin.com/company/apolloio",
                 "twitter_url": "https://twitter.com/MeetApollo/",
                 "facebook_url": "https://www.facebook.com/MeetApollo/",
+                "primary_phone": {
+                    "number": "(202) 374-XXXX",
+                    "source": "Account"
+                },
                 "languages": [],
-                "alexa_ranking": 77520,
-                "phone": null,
-                "linkedin_uid": "18511550",
+                "alexa_ranking": 685,
+                "phone": "(123) 456-XXXX",
+                "linkedin_uid": "185115XX",
+                "founded_year": 2015,
                 "publicly_traded_symbol": null,
                 "publicly_traded_exchange": null,
-                "logo_url": "https://apollo-server.com/uploads/pictures/5f0265XXXXXXXXXXXXXXXXXX/picture",
+                "logo_url": "https://zenprospect-production.s3.amazonaws.com/uploads/pictures/6188c0967ae1c20001869130/picture",
```

Comment:
@spal-apollo 
Can we not show the id here 6188c0967ae1c20001869130 => 61XXc0X67XXXXXXXXXXXXXXX

also, can we avoid `https://zenprospect-production.s3.amazonaws.com/` being in the documentation and leave as `https://apollo-server.com/uploads/pictures/`